### ISSUE TITLE
[Feature Request] OLED display editor

### ISSUE THREAD

**Issue by z4rathustr4** [the-via/keyboards#2110]
Hey there, I know you're taking a vacation from PRs, I would do it too. Just wanna know if you guys have any plans on adding an OLED display editor for CRKBD keyboards. I know you can do it compiling the whole QMK crkbd firmware, but I've just bought my first Corne split keyboard and I'm worried to break the firmware while flashing.

Resources I've found:
* [https://www.reddit.com/r/MechanicalKeyboards/comments/gjejxi/qmk_logo_editor/](https://www.reddit.com/r/MechanicalKeyboards/comments/gjejxi/qmk_logo_editor/)
* [https://github.com/joric/qle](https://github.com/joric/qle)
* QMK docs on OLED: [https://docs.qmk.fm/#/feature_oled_driver?id=oled-api](https://docs.qmk.fm/#/feature_oled_driver?id=oled-api)

Have a nice winter festivity/vacation!

**Comment by Xelus22** [comment]
Hi, thanks for the feature request.

Unfortuntately this request would require a huge change in how VIA GUI interacts with QMK. Currently all OLED animations are "baked-in" to the generated binary in QMK. This means you can't change it after generating it.
To create an editor in VIA, theres a lot of work to move that to EEPROM, which would take a significant chunk of EEPROM and require a much larger effort from both QMK and VIA teams to make this happen. 

So TL;DR, could be a thing but *very* far in the future.

**Comment by z4rathustr4** [comment]
Take your time, I just wish in the future being able to modify the OLED screen intuitively. 

Thanks for answering. Have a nice day! 😄
